scripts/cat/save_load.py

```diff
@@ -0,0 +1,151 @@
+import os
+from typing import TYPE_CHECKING, Type
+
+import ujson
+
+from scripts.game_structure.game.save_load import safe_save
+from scripts.game_structure.game.settings.settings import get_game_setting
+from scripts.housekeeping.datadir import get_save_dir
+
+if TYPE_CHECKING:
+    from scripts.cat.cats import Cat
+    from scripts.game_structure.game_essentials import Game
+
+faded_ids = []
+"""List of IDs of faded cats"""
+
+cat_to_fade = []
+"""Cats who have been faded since the last save"""
+
+
+def save_cats(clanname, cat_class: Type["Cat"], game: "Game"):
+    """Save the cat data."""
+
+    directory = get_save_dir() + "/" + clanname
```

Comment:
Moving forward, we should try to use Pathlib instead of constructing path literals as strings.